_component_design/chips.md

```diff
@@ -52,9 +52,13 @@ Chips are best used with indicating a list of items that may be part of a differ
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Default chip
+## Default chip
+
+The default chip is a single circle that contains a preview of the item the chip represents. The preview can consist of the following:
 
-The default chip is a single circle that contains a preview of the item the chip represents. The preview can consist of the the following; letters to indicate the name, an image to indicate a user, or a product icon to indicate the type of device.
+- letters to indicate the name
+- an image to indicate a user
+- a product icon to indicate the type of device
```

Comment:
Just asking for a gut-check; Should list items be sentence fragments without `.`'s? @nimbinatus

Reply:
List items with sentence fragments do not have periods unless any of the fragments is followed by a complete sentence, in which case all fragments end in a period.

What is incorrect in the list you point out is that I neglected to capitalize the first letter. I have made that correction.